### DISCUSSION TITLE
Make ready sound effect clientside

### DISCUSSION THREAD

**Discussion by StartingTimer** [ppy/osu#17608] [Ideas] [answered]
In multiplayer lobbies people can spam ready and unready and it sends a sound to everyone in the lobby which is annoying.

**Comment by StartingTimer** [comment, marked as answer]
sorry i didnt realise this was a duplicate and i can't delete it

**Comment by peppy** [comment]
It's intentional that everyone can hear these, but based on feedback we may debounce on a per player basis to reduce the spamminess.
